logic/poi_locations.yaml

```diff
@@ -110,7 +110,7 @@
       top: 0
   logic: westOfMire
 - name: DW Death Mountain
-  hoverText: DW Death Mountain
+  hoverText: "DW Death Mountain (2) : Don't need <img src='/static/image/moonpearl.png' class='mini'>"
```

Comment:
Why did you add double quotes on this one?

Reply:
The colon in the `hoverText` confuses YAML parsers if the string isn't quoted.